tests/test_helpers/test_exec_timeout.py

```diff
@@ -56,7 +56,7 @@ def setUpClass(cls):
     def test_cancel_by_timeout(self):
         """Test function interrupted by timeout."""
         slow_function_time = 0.4
-        timeout = 0.1
+        timeout, margin = 0.1, 0.1
```

Comment:
why is this acceptable? we already assert `exec_timeout.is_cancelled_by_timeout()`
https://github.com/valory-xyz/open-aea/blob/e750ad9fc46f794803df09f0bdfa1da81e7a4155/tests/test_helpers/test_exec_timeout.py#L56-L70